dev-python/PyQt6_sip/PyQt6_sip-13.3.1.ebuild

```diff
@@ -0,0 +1,16 @@
+# Copyright 2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DISTUTILS_USE_PEP517=setuptools
+PYTHON_COMPAT=( python3_{8..10} )
+inherit distutils-r1
+
+DESCRIPTION="sip module support for PyQt6"
+HOMEPAGE="https://www.riverbankcomputing.com/software/sip/"
```

Comment:
Please add the pypi project to HOMEPAGE, and the source repo if you have it too, e.g.:

```
HOMEPAGE="
    https://frobnicate.org/
    https://github.com/whatever/frobnicate/
    https://pypi.org/project/frobnicate/
"
```

It makes it much easier to quickly locate stuff.

Reply:
I tend not to like that because it's redunant on p.g.o. which already shows pypi/github from metadata.xml